src/SprykerEco/Zed/AfterPay/Communication/Plugin/Oms/Condition/IsAuthorizationCompletedPlugin.php

```diff
@@ -24,6 +24,8 @@ class IsAuthorizationCompletedPlugin extends AbstractPlugin implements Condition
     public const AUTHORIZE_TRANSACTION_ACCEPTED = AfterPayConfig::API_TRANSACTION_OUTCOME_ACCEPTED;
 
     /**
+     * {@inheritDoc}
+     *
```

Comment:
Add a specification to a method:
- Checks if the authorization operation was successfully accepted for the order.